imagetool/src/main/java/com/oracle/weblogic/imagetool/api/model/CachedFile.java

```diff
@@ -11,6 +11,7 @@
 import java.nio.file.Paths;
 
 import com.oracle.weblogic.imagetool.cachestore.CacheStore;
+import com.oracle.weblogic.imagetool.installer.InstallerType;
```

Comment:
new copyright rules - see http://pdb.us.oracle.com/downloads/legal_notices.html
when you update a file now, you need to remove the "all rights reserved" from line 1

Reply:
Noted.  Will address with a bulk change, later.